### DISCUSSION TITLE
Anti-cheat mods....

### DISCUSSION THREAD

**Discussion by Snayjay** [Astropilot/ValheimTooler#41] [Q&A] [answered]
How do you stop people from using mods like Valheim Tooler and Quality of Life.  If you make a great world, someone can come in with these mods and literally destroy it in minutes.  I know you can backup your world but that is not the point.  Are there any mods you can load that can keep the use of mods like these from running on a server?

**Comment by Astropilot** [comment, marked as answer]
Hi,

Yes, there is: https://thunderstore.io/c/valheim/p/Azumatt/AzuAntiCheat/
It is known for detecting the most obvious cheats and should detect ValheimTooler and QoL without much difficulty. And it works under a whitelist system, authorizing only a list of desired mods.

Kind regards,
Astropilot

**Comment by Snayjay** [comment]
Appreciate it.  I've got it installed.  Thanks!

Sent via the Samsung Galaxy Z Fold5, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
